### PR TITLE
Updated package version and changelog

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Pre release branch for CENNZnet API. 
Updated package version to 1.3.0-alpha.1 which is smaller than 1.3.0
Updated changelog

### CODE REVIEW

File: packages/api/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,20 @@ All notable changes to this project will be documented in this file.
...
+### Added:
+    - bump http-proxy from 1.18.0 to 1.18.1
+    - Add the publish CircleCi job for the release branches
+### Changed:
+    - Updated the code base to use the current latest polkadot dependencies (^2.3.1 as of now) 
+    - Used eslint over tslint
+### Removed:
+    - Removed wallet support and all crml packages
+    - Removed doughnut support
+    - Removed staking bonding
```
Comment (holygits) [inline packages/api/CHANGELOG.md:17]: this is not an api related change right? ` - Removed staking bonding`

File: packages/api/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,20 @@ All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
 ## [UNRELEASED]
+## [1.3.0-alpha.1](https://github.com/cennznet/api.js/compare/release/1.2.1...prerelease/1.3.0-alpha.1) (23/10/2020)
+### Added:
+    - bump http-proxy from 1.18.0 to 1.18.1
+    - Add the publish CircleCi job for the release branches
```
Comment (holygits) [inline packages/api/CHANGELOG.md:10]: `Add the publish CircleCi job for the release branches` - doesn't need to be in release notes

File: packages/api/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,20 @@ All notable changes to this project will be documented in this file.
...
+### Changed:
+    - Updated the code base to use the current latest polkadot dependencies (^2.3.1 as of now)
```
Comment (holygits) [inline packages/api/CHANGELOG.md:12]: is it easy/hard to pull out some of the breaking changes from the polkadot changelog to put here?

File: packages/api/CHANGELOG.md (hunk 1 above)
Comment (KarishmaBothara) [reply packages/api/CHANGELOG.md:17]: ya.. not related to API.

File: packages/api/CHANGELOG.md (hunk 3 above)
Comment (KarishmaBothara) [reply packages/api/CHANGELOG.md:12]: will check..

File: packages/api/CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,19 @@ All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
 ## [UNRELEASED]
+## [1.3.0-alpha.1](https://github.com/cennznet/api.js/compare/release/1.2.1...prerelease/1.3.0-alpha.1) (23/10/2020)
+Polkadot's changelog at [2.3.1](https://github.com/polkadot-js/api/blob/master/CHANGELOG.md#231-oct-19-2020)
```
Comment (alexsednz) [inline packages/api/CHANGELOG.md:8]: Perhaps we need to mention the range of polkadot changelogs. Something like: [1.1.1-2.3.1]

Comment (KarishmaBothara) [reply packages/api/CHANGELOG.md:8]: We plan to have [how to migrate to 1.3.x](url) page before the actual release which will talk more about things that have changed from 1.1.1 to 2.3.1

**Review by holygits — APPROVED** [review]

**Review by alexsednz — APPROVED** [review]

**Comment by holygits** [comment]
ready in #200

**Pull request closed without merging**